AdminApi/src/AdminApi/ClientApp/src/app/services/identity-service/identity.service.ts

```diff
@@ -26,6 +26,39 @@ export class IdentityService {
         this.odataUrl = `${environment.odataUrl}/Identities`;
     }
 
+    /* eslint-disable @typescript-eslint/naming-convention */
+    public readonly messageTemplates: Record<string, string> = {
+        StartedByOwner: "The deletion process was started by the owner. It was automatically approved.",
+        StartedBySupport: "The deletion process was started by support. It is now waiting for approval.",
+        Approved: "The deletion process was approved.",
+        Rejected: "The deletion process was rejected.",
+        CancelledByOwner: "The deletion process was cancelled by the owner of the identity.",
+        CancelledBySupport: "The deletion process was cancelled by a support employee.",
+        CancelledAutomatically: "The deletion process was cancelled automatically, because it wasn't approved by the owner within the approval period.",
+        ApprovalReminder1Sent: "The first approval reminder notification has been sent.",
+        ApprovalReminder2Sent: "The second approval reminder notification has been sent.",
+        ApprovalReminder3Sent: "The third approval reminder notification has been sent.",
+        GracePeriodReminder1Sent: "The first grace period reminder notification has been sent.",
+        GracePeriodReminder2Sent: "The second grace period reminder notification has been sent.",
+        GracePeriodReminder3Sent: "The third grace period reminder notification has been sent.",
+        DataDeleted: "All {aggregateType} have been deleted."
+    };
+
+    public getFormattedMessage(messageKey: string, additionalData: Record<string, string>): string {
```

Comment:
The class is called `IdentityService`. The method is called `getFormattedMessage`. As the caller, how would you know what this method does? It might be an exception message. Or an error message to display in the UI. Or a message to be sent as an email to someone. You need to be more specific.